packages/rpc-methods/src/restricted/manageState.ts

```diff
@@ -87,6 +92,8 @@ export enum ManageStateOperation {
   updateState = 'update',
 }
 
+export const STORAGE_SIZE_LIMIT = 10000; // In bytes
```

Comment:
We should decide with the team what this value should be.

Reply:
This is now limited with 100MB in size, as discussed on standup call.